Formula/wine.rb

```diff
@@ -38,6 +38,9 @@ def require_universal_deps?
     MacOS.prefer_64_bit?
   end
 
+  option "with-win64",
+         "Build with 64-bit support (only works with Xcode 8 and newer.)"
```

Comment:
Probably worth keeping this on the same line. Make `only works with Xcode 8 and newer.` a `depends_on :xcode => ["8.0", :build] if build.with? "win64"` instead. Add `(won't run 32-bit binaries.)`